src/replica/replica_stub.cpp

```diff
@@ -573,6 +590,35 @@ void replica_stub::initialize(bool clear /* = false*/)
     _access_controller = std::make_unique<dsn::security::access_controller>();
 }
 
+dsn::error_s store_kms_key(std::string data_dir,
```

Comment:
Different from Java, C++ passes a copy of value for parameters, so it's better to use const reference in this case.

Reply:
The other parameters are the same.